src/facades/ServerApi.ts

```diff
@@ -0,0 +1,45 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import { useCallback } from "react";
+
+const API_URL = "https://example.com/api/";
```

Comment:
@EdenGottlieb This should be in an env var probably? I'm guessing we want to add .env for this

Reply:
of course, this is just an example, it should be injected via env